health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -2638,4 +2713,259 @@ export const UICustomizations = {
+                    onClick={() => {
+                      if (isTag) {
+                        setTag("");
+                        setOpenPopUp(false);
+                        setIsTag(false);
+                      } else {
+                        setIsTag(true);
+                      }
+                      return;
+                    }}
+                  />,
+                  <ButtonNew
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)